packages/widgets/package.json

```diff
@@ -10,20 +10,23 @@
   ],
   "scripts": {
```

Comment:
Kindly update the version of the package.

Reply:
Already updated